src/sentry/migrations/0294_backfill_alert_owners.py

```diff
@@ -0,0 +1,66 @@
+# Generated by Django 2.2.27 on 2022-04-08 22:24
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_alert_owners(apps, schema_editor):
+    AlertRule = apps.get_model("sentry", "AlertRule")
+    OrganizationMember = apps.get_model("sentry", "OrganizationMember")
+    User = apps.get_model("sentry", "User")
+    Team = apps.get_model("sentry", "Team")
+
+    for alert_rule in RangeQuerySetWrapperWithProgressBar(AlertRule.objects_with_snapshots.all()):
```

Comment:
Let's add `.select_related("owner")`. This saves us making an additional db call when we access `alert_rule.owner`